packages/server/graphql/mutations/helpers/addAIGeneratedContentToThreads.ts

```diff
@@ -18,41 +32,31 @@ const addAIGeneratedContentToThreads = async (
     dataLoader.get('teams').loadNonNull(teamId)
   ])
   const {tier} = team
-  const commentPromises = stages.map(async (stage, idx) => {
-    const group = groups.find((group) => group.id === stage.reflectionGroupId)
+  const commentPromises = stages.map(async ({discussionId, reflectionGroupId}, idx) => {
+    const group = groups.find((group) => group.id === reflectionGroupId)
     if (!group?.summary && !group?.discussionPromptQuestion) return
-    const topicSummaryExplainerText =
-      tier === 'starter' ? AIExplainer.STARTER : AIExplainer.PREMIUM_REFLECTIONS
-    const topicSummaryHtml =
-      idx === 0
-        ? `<html><body><i>${topicSummaryExplainerText}</i><br><p><b>🤖 Topic Summary</b></p><p>${group.summary}</p></body></html>`
-        : `<html><body><p><b>🤖 Topic Summary</b></p><p>${group.summary}</p></body></html>`
-    const summaryBlock = convertHtmlToTaskContent(topicSummaryHtml)
-    const topicSummaryCommentInput = {
-      discussionId: stage.discussionId,
-      content: summaryBlock,
-      threadSortOrder: 0,
-      createdBy: PARABOL_AI_USER_ID
-    }
-    const topicSummaryComment = new Comment(topicSummaryCommentInput)
+    const comments: Comment[] = []
 
-    const discussionPromptQuestionHtml =
-      idx === 0
-        ? `<html><body><p><b>🤖 Discussion Question</b></p><p>${group.discussionPromptQuestion}</p></body></html>`
-        : `<html><body><p><b>🤖 Discussion Question</b></p><p>${group.discussionPromptQuestion}</p></body></html>`
-    const discussionPromptQuestionBlock = convertHtmlToTaskContent(discussionPromptQuestionHtml)
-    const discussionPromptQuestionCommentInput = {
-      discussionId: stage.discussionId,
-      content: discussionPromptQuestionBlock,
-      threadSortOrder: 1, // make sure it's after the topic summary
-      createdBy: PARABOL_AI_USER_ID
+    if (group.summary) {
+      const topicSummaryExplainerText =
+        tier === 'starter' && idx === 0 ? AIExplainer.STARTER : AIExplainer.PREMIUM_REFLECTIONS
```

Comment:
-1 Now we're showing the explainer each time, but the first stage sees the starter one for starter
```ts
idx === 0 ? (tier === 'starter' ? AIExplainer.STARTER : AIExplainer.PREMIUM_REFLECTIONS) : null
```